sklearn/manifold/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .mds import MDS
 from .spectral_embedding_ import SpectralEmbedding, spectral_embedding
 from .t_sne import TSNE
+from .laplacian_eigenmap_ import LaplacianEigenmap, laplacian_eigenmap
 
 __all__ = ['locally_linear_embedding', 'LocallyLinearEmbedding', 'Isomap',
-           'MDS', 'SpectralEmbedding', 'spectral_embedding', "TSNE"]
```

Comment:
This will only prevent people from importing both `SpectralEmbedding` and `LaplacianEmbedding` when people do `from sklearn.manifold import *`... People can still do `from sklearn.manifold import SpectralEmbedding`